src/components/Loader/PeopleList.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from './PersonLink';
+import { NO_PARENT } from '../../utils/constants';
+
+type Props = {
+  person: Person,
+  peopleSlug: string | undefined,
```

Comment:
```suggestion
  peopleSlug?: string,
```
You can make the field optional just by adding a question mark